backend/main-service/src/main/java/kkakka/mainservice/coupon/domain/repository/MemberCouponRepository.java

```diff
@@ -28,4 +28,7 @@ Long countByCouponIdAndMemberId(@Param(value = "couponId") Long couponId,
     @Query("select mc from MemberCoupon mc where mc.coupon.priceRule = 'GRADE_COUPON' and mc.member.id = :memberId and mc.isUsed = false")
     List<MemberCoupon> findGradeCouponByMemberId(@Param(value = "memberId") Long memberId);
 
+    @Query("select count(mc) from MemberCoupon mc where mc.member.id = :memberId and mc.isUsed = false")
+    Long countByMemberId(@Param(value = "memberId") Long memberId);
```

Comment:
```suggestion
    int countAllByMemberIdAndIsUsedFalse(@Param(value = "memberId") Long memberId);
```
쿼리 작성하지 않고 이렇게 named query 로 찾을 수 있어요!

Reply:
좋은 정보 감사합니다 👍🏻 이래서 jpa 쓰나봐요~